### PR TITLE
Redo script to support multiple repos

### PR DESCRIPTION
**Pull request by spiffxp**
Pull in images from gcr.io, quay.io/samsung_ag, quay.io/coreos,
prom/promdash

### CODE REVIEW

**Comment by l337ch** [comment]
Moving conversation to here from hipchat :).
I'm sure what this script is supposed to do really. @mikeln ?

**Comment by l337ch** [comment]
i know that the local registry to use for the various demo projects is 172.16.16.15:5000 but i don't know what the scheme should be beyond that. 
I think from the script it should 172.16.16.15:5000/<image>
oh @mikeln i don't want to break anything

File: scripts/pullimages.sh [hunk 1]
```diff
@@ -4,24 +4,40 @@
...
-for IMAGE in ${IMAGES[@]}; do
-   echo "Pulling: ${SOURCE_REPO}/${IMAGE} to ${DEST_REPO}"
-   docker pull ${SOURCE_REPO}/${IMAGE}
-   docker tag  -f ${SOURCE_REPO}/${IMAGE} ${DEST_REPO}/${IMAGE}
-   docker images ${DEST_REPO}/*
-   docker push ${DEST_REPO}/${IMAGE}
+for source_image in ${images[@]}; do
+   repo=$(echo $source_image | cut -d/ -f1)
+   org=$(echo $source_image | cut -d/ -f2)
+   image=$(echo $source_image | sed -e 's,.*/\([^:]*\).*,\1,')
```
Comment (mikeln) [inline scripts/pullimages.sh]: Though I love a good sed arg... Almost rather see something easier to grok.   Echo $source_image | rev | cut -d/ -f1 | rev
Then split that into two at the colon if you must.

File: scripts/pullimages.sh [hunk 2]
```diff
@@ -4,24 +4,40 @@
...
+   tag=$( (echo $source_image | grep -q : && echo $source_image | cut -d: -f2) || echo "latest")
```
Comment (mikeln) [inline scripts/pullimages.sh]: Not sure why you would have an image without a version??

**Comment by mikeln** [comment]
Given that, none of these bug me too much

**Pull request merged**